src/components/ManageUsersModal.jsx

```diff
@@ -102,7 +113,8 @@ export class ManageUsersModal extends React.PureComponent {
         variant="outlined"
         onClick={this.handleClickOpen}
       >
-        {modalText}
+        {/* <Edit fontStyle="small" className={classes.openButtonIcon} /> {modalText} */}
+        <i className={`${classes.openButtonIcon} fa-solid fa-pen-circle`} /> {modalText}
```

Comment:
This doesn't look like a blue circle with a white pencil in the attached screenshots.  Are the screenshots old or is this because DR-2735 is still pending?

Reply:
It's because that PR wasnt merged when I made this pr! :) I can update it now!

Reply:
Or actually wait, sorry, I already updated it. The screenshots are old, but the code is the pencil circle!